src/components/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,69 @@
+import { useSearchParams } from 'react-router-dom';
+import { useMemo } from 'react';
+import { PeopleFilters } from '../PeopleFilters';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+import { usePeople } from '../../store/peopleContext';
+
+export const PeoplePage = () => {
+  const {
+    people, loading, loadingError, filterPeople,
+  } = usePeople();
```

Comment:
do you really need Context in this app? Looks like you use it only in one place.
Here is the [article](https://blog.logrocket.com/pitfalls-of-overusing-react-context/) about Context overusing. Please read it